.env

```diff
@@ -0,0 +1,4 @@
+APP_ENV=dev
+APP_SECRET=ch4n63m31fy0uc4n
+TRUSTED_PROXIES=127.0.0.1/8,10.0.0.0/8,172.16.0.0/12,172.17.0.0/16,172.18.0.0/15,172.20.0.0/14,172.24.0.0/13,192.168.0.0/16
```

Comment:
original value is `127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16`
updated to add more based on [this article](https://jolicode.com/blog/symfony-reverse-proxies-and-ip-protection#reverse-proxies-in-docker)